src/ssr-template-helper.js

```diff
@@ -96,17 +98,22 @@ export class SsrTemplateHelper {
 
   /**
    * @param {!Element} element
-   * @param {!JsonObject} data
+   * @param {!Array|!JsonObject|string} data
```

Comment:
Related to above comment -- doesn't look like we handle a `string` case in this code.

Reply:
right, we are no longer accepting string. good find. fixed.